src/oas.md

```diff
@@ -3284,6 +3341,229 @@ animals:
 </aliens>
 ```
 
+###### Elements With Attributes And Text
+
+```yaml
+properties:
+  animals:
+    type: array
+    xml:
+      nodeType: element
+      name: animals
+    items:
+      properties:
+        kind:
+          type: string
+          xml:
+            nodeType: attribute
+            name: animal
```

Comment:
```suggestion
    items:
      xml:
        name: animal
      properties:
        kind:
          type: string
          xml:
            nodeType: attribute
```
I think the items schema needs an xml field to set the name of the XML elements corresponding to the items.